### PR TITLE
Return TS Errors

### LINKED ISSUE #606: Return Transmittal Sheet edits in `/edits` API endpoint
Depends on #605

### PR DESCRIPTION
**Pull request by nickgrippin**
Depends on #617 

Closes #606

### CODE REVIEW

**Comment by nickgrippin** [comment]
The build is failing due to a timeout issue that's fixed in another PR.

File: api/src/test/scala/hmda/api/http/ValidationErrorConverterSpec.scala [hunk 1]
```diff
@@ -17,17 +17,17 @@ class ValidationErrorConverterSpec extends WordSpec with MustMatchers with Valid
       val ctx = ValidationContext(None, Some(2017))
       val errors = badLars.flatMap(lar => validationErrors(lar, ctx, validateLar).errors)
       val syntacticalEditResults =
-        validationErrorsToEditResults(errors, Syntactical)
+        validationErrorsToEditResults(Nil, errors, Syntactical)
```
Comment (schbetsy) [inline api/src/test/scala/hmda/api/http/ValidationErrorConverterSpec.scala]: With this update, this spec file needs an additional test to show that Transmittal Sheet errors convert correctly.

File: api/src/test/scala/hmda/api/http/institutions/SubmissionPathsSpec.scala [hunk 2]
```diff
@@ -102,8 +102,8 @@ class SubmissionPathsSpec extends InstitutionHttpApiSpec {
     val expectedEdits =
       EditResults(
         List(
-          EditResult("V285", List(LarEditResult(LarId("loan2")), LarEditResult(LarId("loan3")))),
-          EditResult("V280", List(LarEditResult(LarId("loan1"))))
+          EditResult("V285", ts = false, List(LarEditResult(LarId("loan2")), LarEditResult(LarId("loan3")))),
```
Comment (schbetsy) [inline api/src/test/scala/hmda/api/http/institutions/SubmissionPathsSpec.scala]: This endpoint also needs a test to demonstrate that TS errors are published correctly

**Comment by nickgrippin** [comment]
@schbetsy: The tests should be updated

**Comment by kgudel** [comment]
@schbetsy and I reviewed this code and it looks good! The only issue that we found were some readability problems in `ValidationErrorConverter`. Neither of us thought that it should block merge but we wanted to bring it up as an issue to address at some point, even if it isn't now.

**Pull request merged**